packages/react/src/useSynthql.test.tsx

```diff
@@ -79,7 +79,7 @@ describe('useSynthql tests', () => {
             first_name: 'PENELOPE',
             last_name: 'GUINESS',
         });
-    }, /* 10 seconds */ 10_000);
```

Comment:
@jimezesinachi pay more attention to your diffs. This is a big sign that you're not reading the code you're pushing.

Reply:
Once you've fixed this, feel free to merge.